contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,727 @@
+    	</tr>
+    	<tr>
+        	<td>A/B testing</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">CRO and test in production</td>
+    	</tr>
+    	<tr>
+        	<td>Group analytics</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Analyze groups (e.g. companies and teams)</td>
+    	</tr>
+    	<tr>
+        	<td>SQL access</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>  
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Write your own queries and filters</td>
+    	</tr>
+    	<tr>
+       	<td>Transparent pricing</td>
+      		<td className="text-center"><span className="text-green text-lg">✔</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Know exactly what you'll pay</td>
+    	</tr>
+    	<tr>
+       	<td>Billing limits</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Prevent surprise bills</td>
+    	</tr>
+    	<tr>
+        	<td>Open source</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Open core, build your own integrations</td>
+   		</tr>
+    	<tr>
+        	<td>Cute hedgehogs</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Joy</td>
+   		</tr>
+	</tbody>
+</table>
+</div>
+
+### Key differences
+
+1. **Autocapture:** PostHog automatically captures events, which means you don't have to manually decide what events to track, unless you want to. It saves time for engineers and product managers, and ensures you'll always have usable data even if you forgot to create a custom event. 
+
+2. **Single tool vs all-in-one:** While it offers integrations with third-party A/B testing, feature flag, and session replay tools, Mixpanel doesn't offer much outside typical product analytics features. PostHog integrates all these features into one platform.
+
+3. **Avoiding surprise bills:** PostHog allows you to set billing limits, so you can avoid surprise bills. Mixpanel charges you for any overage above your pre-paid events volume.
```

Comment:
This feels a bit weak as a difference. It's not really a PostHog selling feature, just a nice-to-have tbh.

Reply:
Cookie-less tracking? Heatmaps, dev tool integrations (like Sentry)?

Reply:
Fair point. Will have a think.